cli/test/e2e/package_repo_release_test.go

```diff
@@ -49,13 +50,47 @@ func TestPackageRepositoryReleaseInteractively(t *testing.T) {
 			promptOutput.Write(env.Image)
 		}()
 
-		kctrl.RunWithOpts([]string{"pkg", "repo", "release", "--tty=true", "--chdir", workingDir, "--version", "1.0.0"},
+		version := "1.0.0"
+		kctrl.RunWithOpts([]string{"pkg", "repo", "release", "--tty=true", "--chdir", workingDir, "--version", version},
 			RunOpts{NoNamespace: true, StdinReader: promptOutput.StringReader(),
 				StdoutWriter: promptOutput.BufferedOutputWriter(), Interactive: true})
 
 		keysToBeIgnored := []string{"creationTimestamp:", "image"}
 		verifyPackageRepoBuild(t, keysToBeIgnored)
 		verifyPackageRepository(t, keysToBeIgnored)
+
+		args := []string{"tag", "list", "-i", os.Getenv("KCTRL_E2E_IMAGE")}
+		cmd := exec.Command("imgpkg", args...)
+		output, err := cmd.Output()
+		// t.Logf("the tags are: %s", output)
```

Comment:
nit
```suggestion
```

Reply:
removed in commit: c65ba79a7092f834b8bb5109b86b7942ef04e58b